content/beam/capability/2016/03/17/capability-matrix.html

```diff
@@ -154,19 +154,23 @@ <h1 class="post-title" itemprop="name headline">Clarifying & Formalizing Runner
     
     
 
-    <td width="25%" class="cap-summary" style="background-color:#ec3;border-color:#ca1"></td>
+    <td width="25%" class="cap-summary" style="background-color:#ec3;border-color:#ca1"><center><b>&#x2713;</b></center>
```

Comment:
@davorbonaci That looks like one of the explicit "fixes" to create valid HTML. We were embedding a `<content>` tag inside a `<b>` tag which is apparently invalid HTML. I swapped them in the following files which I suspect might be the reason for that update:

- `_includes/capability-matrix-row-blog.md`
- `_includes/capability-matrix-row-full.md`
- `_includes/capability-matrix-row-summary.md`